data-structures-linked-lists/get-length.js

```diff
@@ -0,0 +1,17 @@
+/* exported getLength */
+
+function getLength(list) {
+
+  let listCopy = list;
```

Comment:
This is not a copy. Objects are pass-by-reference. We don't really need a copy anyway.